src/include/stir/listmode/ListModeData.h

```diff
@@ -238,6 +238,41 @@ class ListModeData : public ExamData
   shared_ptr<const ProjDataInfo> proj_data_info_sptr;
 };
 
+
+class ListModeData_dummy : public ListModeData
```

Comment:
I'm not sure why you need this, but if you do, can you create a minimal doxygen header, saying that this is a temporary class that should not be used

Reply:
Also, correct grammar in messages below "does not have"

Reply:
I can not figure a way around it without changing way too much. 
If I change get_input_data() to return ExamData() the set_up breaks. 
If I store the ExamInfo separately, I have to change the set_up() of all base classes. 

The only alternative I could think of was to set a listmode file of another format, but then I would have to replace everything manually.

Reply:
It is very tricky.

Reply:
I added a brief doxygen header.